scripts/pre_commit_linter.py

```diff
@@ -569,10 +568,15 @@ def _check_bad_pattern_in_file(filename, content, pattern):
     if not (any(filename.startswith(excluded_dir)
                 for excluded_dir in pattern['excluded_dirs'])
             or filename in pattern['excluded_files']):
-        if re.search(regexp, content, re.MULTILINE):
-            print '%s --> %s' % (
-                filename, pattern['message'])
-            return True
+        bad_pattern_count = 0
+        with open(filename, 'r') as file_to_check:
+            for line_num, line in enumerate(file_to_check, 1):
+                if re.search(regexp, line):
+                    print 'L: %s %s --> %s' % (
```

Comment:
@seanlip: I have added a detailed error message which will help us while debugging:  
`L: <line_number> <filename> --> There should not be any trailing whitespaces.`  
Thanks!